### DISCUSSION TITLE
Integrate PuTTY into it?

### DISCUSSION THREAD

**Discussion by ghost** [GitSquared/edex-ui#961] [Questions] [answered]
I use putty a lot at work in windows.

But edexUI in windows mean only the crappy MsDos... zzz

So, i want to suggest somebody to make it work with PuTTY.

Ok thanks.

**Comment by GitSquared** [comment, marked as answer]
See #958 for how to change the default shell. Just replace `powershell.exe` with `putty.exe` in the settings.
